src/main/java/uk/gov/hmcts/reform/divorce/casemaintenanceservice/config/HttpConnectionConfiguration.java

```diff
@@ -86,7 +85,6 @@ private ClientHttpRequestFactory getClientHttpRequestFactory(
         CloseableHttpClient client = HttpClientBuilder
             .create()
             .useSystemProperties()
-            .addInterceptorFirst(new OutboundRequestIdSettingInterceptor())
```

Comment:
I thought we need this for logging request id by the java logging library https://github.com/hmcts/java-logging/blob/master/java-logging-httpcomponents/src/main/java/uk/gov/hmcts/reform/logging/httpcomponents/OutboundRequestIdSettingInterceptor.java#L25 🤔

Reply:
I checked with the tech leads and Tim said it was before AppInsights. There was a custom elk logging stack